src/main/java/com/walking/intensive/chapter2/task8/Task8.java

```diff
@@ -19,12 +21,63 @@
  * <p><a href="https://github.com/KFalcon2022/intensive-tasks-2024/blob/master/README.md">Требования к оформлению</a>
  */
 public class Task8 {
+
     public static void main(String[] args) {
-//        Для собственных проверок можете делать любые изменения в этом методе
+        int[] sumFrequency = new int[28];
+
+        for (int i = 0; i < 1000 ; i++) {
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter2/task8/Task8.java:28:34: ';' is preceded with whitespace. [NoWhitespaceBefore]